components/ActionSheet/examples/ActionSheetSwiftExampleViewController.swift

```diff
@@ -13,21 +13,20 @@
 // limitations under the License.
 
 import UIKit
-import MaterialComponentsBeta.MaterialActionSheet
-import MaterialComponentsBeta.MaterialActionSheet_Theming
+
 import MaterialComponents.MaterialColorScheme
 import MaterialComponents.MaterialTypographyScheme
 
+import MaterialComponentsBeta.MaterialActionSheet
+import MaterialComponentsBeta.MaterialActionSheet_Theming
+import MaterialComponentsBeta.MaterialContainerScheme
+
 class ActionSheetSwiftExampleViewController: UIViewController {
 
   var colorScheme = MDCSemanticColorScheme()
   var typographyScheme = MDCTypographyScheme()
-  var containerScheme: MDCContainerScheming {
-    let scheme = MDCContainerScheme()
-    scheme.colorScheme = colorScheme
-    scheme.typographyScheme = typographyScheme
-    return scheme
-  }
+  var containerScheme = MDCContainerScheme()
```

Comment:
Is this a work in progress? I don't see any use of `containerScheme` in the example code.  Please also remove the Color and Typography schemes.

Reply:
We were already using containerScheme this just cleaned up creating it.

https://github.com/material-components/material-components-ios/blob/0fa43168923240abd504923125e80b9775d39390/components/ActionSheet/examples/ActionSheetSwiftExampleViewController.swift#L95

Reply:
Ah, sorry. I was looking for `self.containerScheme`. I forgot Swift doesn't use self references.